src/Uno.Sdk/targets/Uno.SingleProject.WinAppSdk.targets

```diff
@@ -51,6 +51,7 @@
 		<_SingleProjectHostArchitecture>$(_SingleProjectHostArchitecture.ToLower())</_SingleProjectHostArchitecture>
 		<Platform>$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)</Platform>
 		<RuntimeIdentifier>win-$(_SingleProjectHostArchitecture)</RuntimeIdentifier>
+		<PublishProfile Condition=" $(PublishProfile) == '' ">win-$(Platform).pubxml</PublishProfile>
```

Comment:
Should this be for executable only?

This also exists in https://github.com/unoplatform/uno/blob/951b222d10c731f747ed9800242cb23b29e658bf/src/Uno.Sdk/targets/Uno.Common.WinAppSdk.targets#L42. Does it have wrong condition in the other file?

Reply:
In the other file it seems to apply to non-singleproject only, added the condition for _IsExecutable